kedro-datasets/pyproject.toml

```
@@ -183,6 +183,7 @@ langchain-chatopenaidataset = ["langchain-openai~=0.1.7"]
 langchain-openaiembeddingsdataset = ["langchain-openai~=0.1.7"]
 langchain-chatanthropicdataset = ["langchain-anthropic~=0.1.13", "langchain-community~=0.2.0"]
 langchain-chatcoheredataset = ["langchain-cohere~=0.1.5", "langchain-community~=0.2.0"]
+langchain-langchainpromptdataset = ["langchain~=0.2.0"]
```

`~=` means compatible with version `0.2.0` that is, `≥0.2.0` and `<0.3.0`
```suggestion
langchain-langchainpromptdataset = ["langchain>=0.2.0"]
```